pyproject.toml

```diff
@@ -36,7 +36,7 @@ itsdangerous = "*"
 jsonschema = "*"
 kombu = "*"
 marshmallow = "*"
-marshmallow-sqlalchemy = "*"
```

Comment:
why are we specifying a version for `marshmallow-sqlalchemy`? Was this just an automatic poetry thing, or is there a reason?

Reply:
Please see #2284.